src/components/VxpAccordion.vue

```diff
@@ -0,0 +1,95 @@
+<template>
+  <StackLayout class="accordion">
```

Comment:
I think css rule should be vxp-accordion

Reply:
replaced css class names.